INS0301-INS0400/INS0329DE.xml

```diff
@@ -76,7 +76,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             mausoleum for the members of the royal family and seems to have been
                             very prosperous. <date notBefore="1563" notAfter="1574">Between 1563 and
                                 1574</date>, <roleName type="title">mamhǝr</roleName>
-                            <persName ref="PRS4283GabraKr">Gabra Krǝstos</persName> restored the
+                            <persName ref="PRS13661GabraKr">Gabra Krǝstos</persName> restored the
```

Comment:
I would restore back the IDs, I do not see the necessity of merge records and create a new one instead, when the IDs were already correct for the two abbots of two monasteries...